compose/ui/ui-uikit/src/uikitMain/objc/CMPUIKitUtils/CMPUIKitUtilsTests/CMPViewControllerTests.swift

```diff
@@ -216,7 +216,16 @@ final class CMPViewControllerTests: XCTestCase {
     }
     
     @MainActor
-    public func testFullscreenPresentationOnTop() async {
+    public func testFullscreenPresentationOnTop() async throws {
+        /*
+         Fails on CI with
+         
+         CMPViewController shouldn't be reused after completely removed from hierarchy,
+         because it's logically marked as Destroyed.
+         You must create a new CMPViewController and use it instead. (CMPViewControllerMisuse)
+         */
+        throw XCTSkip("TODO: Investigate CI fail")
```

Comment:
FYI it was flaky before, but now CI fails always (3 times in row). Locally it's fine. Agreed with @elijah-semyonov to temporary disable it